src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -749,6 +749,16 @@ std::optional<winrt::Microsoft::Terminal::Settings::Model::LaunchMode> AppComman
 // - 0 if the commandline was successfully parsed
 int AppCommandlineArgs::ParseArgs(winrt::array_view<const winrt::hstring>& args)
 {
+    for (auto& arg : args)
+    {
+        if (arg == L"-Embedding")
+        {
+            winrt::Microsoft::Terminal::Settings::Model::ActionAndArgs embeddingAction(ShortcutAction::ToggleInboundPty, nullptr);

```

Comment:
Oh so we're using this as kinda a "if this arg is present, then just skip _everything_ else, don't even bother with the rest of the parsing" thing. I guess that makes sense